06_modules_lab/02.py

```diff
@@ -1,4 +1,21 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-"""
-
+"""
+write a program eivh gets in argv 2 numbers and prints its sum
+
+
+
+"""
+
+import sys
+
+if len(sys.argv) != 3:
+	sys.exit("please enter exactly 2 parameters")

```

Comment:
No need for this check. If length is different the program will fail later on and the next except will catch it (giving you the same result with a few less lines of code).